src/pages/AddBeerPage.jsx

```diff
@@ -0,0 +1,87 @@
+import axios from "axios";
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+function AddBeerPage() {
+  const navigate = useNavigate();
+
+  const [name, setName] = useState("");
+  const [tagline, setTagline] = useState("");
+  const [description, setDescription] = useState("");
+  const [first_brewed, setFirst_brewed] = useState("");
+  const [brewers_tips, setBrewers_tips] = useState("");
+  const [attenuation_level, setAttenuation_level] = useState();
+  const [contributed_by, setContributed_by] = useState("");
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const newBeer = {
+      name,
+      tagline,
+      description,
+      first_brewed,
+      brewers_tips,
+      attenuation_level,
+      contributed_by,
+    };
+
+    axios
+      .post("https://ih-beers-api2.herokuapp.com/beers/new", newBeer)
+      .then((result) => {
+        navigate(`/beers`);
+      })
+      .catch((err) => {});
+  };
+
+  return (
+    <div>
+      <form onSubmit={handleSubmit}>
+        <input
```

Comment:
I would prefer to have a label here together with the `input` so it would add more readability for screen readers :)